tests/load/pipeline/test_drop.py

```diff
@@ -167,7 +167,7 @@ def test_drop_command_resources_and_state(
     if not in_source:
         source = list(source.selected_resources.values())
 
-    pipeline = destination_config.setup_pipeline("droppable", dev_mode=True)
+    pipeline = destination_config.setup_pipeline("droppable")
```

Comment:
would be good to know why you remove this flag here. our change should be not visible in most of the cases